test/lib/mayaUsd/fileio/testImportChaser.py

```diff
@@ -67,6 +67,24 @@ def PostImport(self, returnPredicate, stage, dagPaths, sdfPaths, jobArgs):
 
             self.editsRecord.append([fnDagNode,strAttr])
 
+        # Get the Sdf path to Dag path matching of all imported prims
+        # Call GetSdfToDagMap() to get the sdfToDagMap, 
+        # which contains SDF path to Dag path mapping
+        print(" ----------------- SdfToDagMap ------------------")
+        allSdfPaths = []
+        allDagPaths = []
+        sdfToDagMap = self.factoryContext.GetSdfToDagMap()
+        for sdfToDag in sdfToDagMap:
+            allSdfPaths.append(sdfToDag.key())
+            allDagPaths.append(sdfToDag.data())
+
+        assert(len(allSdfPaths) == len(allDagPaths))
+
+        for i in range(len(allSdfPaths)):
+            print(("----------primitive ") + str(i))
+            print(str(allSdfPaths[i]))
+            print(str(allDagPaths[i]))
```

Comment:
I don't think you are actually testing anything here. You need to have the code that asserts, and not just printing. You can actually remove printing as this isn't very useful.